modules/persistent-storage-csi-manila-install-driver.adoc

```diff
@@ -93,11 +94,12 @@ replicaset.apps/openstack-manila-csi-controllerplugin-7d4f5d985b  1       1
 
 . Verify that the storage class was created successfully by entering the following command:
 +
+[source,terminal]
 ----
 $ oc get storageclasses | grep -E "NAME|csi-manila-"
 ----
 +
-Example output:
+.Example output
```

Comment:
Should there also be a `[source,terminal]` here?

Reply:
Yes, good catch! Resolved.